ui/components/ui/currency-display/currency-display.component.js

```diff
@@ -27,25 +28,30 @@ export default function CurrencyDisplay({
     denomination,
     currency,
     suffix,
+    type,
   });
   return (
-    <div
-      className={classnames('currency-display-component', className)}
-      data-testid={dataTestId}
-      style={style}
-      title={(!hideTitle && title) || null}
-    >
-      {prefixComponent}
-      <span className="currency-display-component__text">
-        {parts.prefix}
-        {parts.value}
-      </span>
-      {parts.suffix && (
-        <span className="currency-display-component__suffix">
-          {parts.suffix}
-        </span>
-      )}
-    </div>
+    <>
```

Comment:
is the fragment here required?

Reply:
In order to enable the entire return body of this component to be conditionally rendered ( per the nullable parts variable discussed below )it appears to be necessary to wrap in a fragment.